### ISSUE TITLE
Structure panel missing members/parse problem for "while"

### ISSUE THREAD

**Issue by jfehrle** [reasonml-editor/reasonml-idea-plugin#189]
plugin version: 0.81

`foo` is missing from the structure panel.  Removing the while...done makes it appear.

![image](https://user-images.githubusercontent.com/1253341/67035195-0d628380-f0ce-11e9-9da5-cef66d3609dc.png)

```
let utf8_length s =
  while !p < len do
    ()
  done;
  ()
  
let foo x = x
```

**Comment by giraud** [comment]
fixed in 0.82

**Comment by jfehrle** [comment]
Also looks good.
